### ISSUE TITLE
सर पर मौत मँडरा रही है

### ISSUE THREAD

**Issue by aryamanarora** [aryamanarora/carmls-hi#19]
From _The Little Prince_:
> सर **पर** मौत मँडरा रही है
> _head **on** death gather-IMPV CONT be-PRS_
> death is closing upon my head/death is threatening me/I am in danger of death

This is highly idiomatic. The verb मँडराना is most commonly literally used to refer to the gathering of clouds, signalling an impending thunderstorm. The figurative situation here is that death is ominous looming, so some kind of a reverse Circumstance (since the object is the participant not the circumstance)? On the other hand, I could just do Goal↝Locus as the literal reading (endpoint of motion).

Also note things like सर पर नशा चढ़ना "to have intoxication rise on the head=>to be intoxicated", सर पर सवार "mounting my head=>occupying my thoughts".

**Comment by aryamanarora** [comment]
So for metaphors we prefer the source domain reading. Goal~Locus is probably best!

**Comment by nitinvwaran** [comment]
TL;DR: Agree that Goal~Locus is probably best.

I found another possible metaphorical usage, from the online Sketch Engine (https://www.sketchengine.eu/) which has the Hindi Web 2012 (HindiWaC v.4) dataset, found using the concordance feature.  The full context is given below, the sentence is a mix of english and hindi:

अगर भैया भाभी अपने 7 फेरों के वचन से 3-4 साल के लिए deviate हुवे. Divorce से तो अच्छी चीज़ है. I know लोग मेरे **सर पर** जुटे बरसायेंगे पर this is practicality यही है. Equality यही है.

Attempting to gloss the relevant part (probably some mistakes, apologies!)

I know लोग मेरे **सर पर** जुटे बरसायेंगे 
I know people me-GENITIVE-PLURAL **head on** converge rain-SIMPLEFUTURE-PLURAL
The जुटे, i guess suggests a GOAL like element, which might support the original conclusion in the thread, of Goal~Locus..

Another that i think also support Goal~Locus, from Sketch Engine..
दूसरे के **सर पर** मढ देना 
another-PLURAL GENITIVEMARKER-PLURAL **head on** mount-STEM do-INFINITIVE
where the action of mounting suggests a destination or Goal...

**Comment by aryamanarora** [comment]
I have a feeling the first one could be a typo for जूते? (As in, they will rain shoes on my head, i.e. insult/deride me). But that still supports Goal\~Locus, again in a metaphorical sense. The second one, I believe, does too. So I'm good with **Goal\~Locus** on this, and I'll close the issue.

The gloss was perfectly understandable! By the way, it's probably useful to check out [common glossing abbreviations](https://en.wikipedia.org/wiki/List_of_glossing_abbreviations). I would probably gloss those two like (dropping some of the gender/number stuff where it's unnecessary):

> I know लोग मेरे सर **पर** जुटे बरसायेंगे
> I know people 1SG.GEN head **on** converge? rain.CAUS.FUT

> दूसरे के सर **पर** मढ देना
> another.OBL GEN.OBL head **on** mount give.INF

(note that देना is a [light verb](https://en.wikipedia.org/wiki/Light_verb) here, and the first two words are actually oblique case and singular)
